reVX/plexos/plexos_plants.py

```diff
@@ -0,0 +1,1006 @@
+        plants_meta : pandas.DataFrame
+            Location (lat, lon), final capacity, and associated sc_gids,
+            res_gids, and res gid_counts for all plants
+        """
+        plants_meta = []
+        for i, plant in enumerate(self.plants):
+            res_gids = plant['res_gids'].values.tolist()
+            plants_meta.append(pd.Series(
+                {'sc_gids': plant['sc_gid'].values.tolist(),
+                 'res_gids': res_gids,
+                 'gid_counts': plant['gid_counts'].values.tolist(),
+                 'gen_gids': [[self.cf_gid_map[gid] for gid in gids]
+                              for gids in res_gids],
+                 'res_cf_means': plant['cf_means'].values.tolist(),
+                 'build_capacity': plant['build_capacity'].values.tolist(),
+                 'cf_mean': np.hstack(plant['cf_means'].values).mean()},
+                name=i))
+
+        plants_meta = pd.concat(plants_meta, axis=1).T
+        plants_meta.index.name = 'plant_id'
+
+        plants_meta = self.plexos_table.merge(plants_meta.reset_index(),
+                                              on='plant_id', how='outer')
+
+        if out_fpath:
+            plants_meta.to_csv(out_fpath, index=False)
+
+        return plants_meta
+
+    def _make_plant_meta(self, bus_meta):
+        """
+        Create plant meta data for given bus
+
+        Parameters
+        ----------
+        bus_meta : pandas.Series
+            Meta data for desired bus to build plant for
+
+        Returns
+        -------
+        plant_meta : pandas.DataFrame
+            Meta data for plant associated with given bus, constructed from:
+            - Plant table
+            - Supply Curve table
+            - Bus capacity
+        """
+        plant_meta = pd.concat(self[bus_meta['plant_id']], axis=1).T
+        plant_meta['gen_gids'] = \
+            plant_meta['res_gids'].apply(lambda gids: [self.cf_gid_map[gid]
+                                                       for gid in gids])
+
+        sc_cols = ['res_gids', 'gen_gids', 'gid_counts', 'capacity']
+        sc_cols = [c for c in self.sc_table if c not in sc_cols]
+        plant_meta = plant_meta.merge(self.sc_table[sc_cols],
+                                      on='sc_gid', how='left')
+
+        plant_capacity = plant_meta['build_capacity'].sum()
+        if plant_capacity != bus_meta['capacity']:
+            bulid_capacity = (plant_meta['build_capacity'] / plant_capacity
+                              * bus_meta['capacity'])
+            plant_meta.loc[:, 'build_capacity'] = bulid_capacity
+
+        return plant_meta
+
+    @staticmethod
+    def _make_profile(cf_fpath, plant_build):
+        """
+        Make generation profiles for given plant buildout
+
+        Parameters
+        ----------
+        cf_fpath : str
+            Path to reV Generation output .h5 file to pull CF profiles from
+        plant_build : pandas.DataFrame
+            DataFrame describing plant buildout
```

Comment:
Can you describe this description :) ?

Is this the list of SC points associated with this plant? With a capacity column maybe?